### DISCUSSION TITLE
ReflectionClass::hasProperty(): Passing null to parameter #1 ($name) of type string is deprecated

### DISCUSSION THREAD

**Discussion by robert-stanciu** [livewire/livewire#8070] [1. Help]
Hi,
I am getting a ton of the following warnings just after I released my Livewire 3 application (Updated from V2).

`ReflectionClass::hasProperty(): Passing null to parameter #1 ($name) of type string is deprecated in /vendor/livewire/livewire/src/Features/SupportQueryString/BaseUrl.php on line 42`

Ubuntu 22.04.1 LTS
PHP 8.2.10
Laravel v10.46.0
Livewire v3.6.4

**Comment by robert-stanciu** [comment]
Anything can be done?

**Comment by joshhanley** [comment]
Incase anyone else comes across this, there is an open PR #8099 from @robert-stanciu for this 🙂
